tests/gpflow/models/test_svgp_posterior.py

```diff
@@ -0,0 +1,77 @@
+from itertools import product
+from typing import Tuple
+
+import numpy as np
+import pytest
+import tensorflow as tf
+
+import gpflow
+from gpflow.models.svgp import NewSVGP, OldSVGP
+
+input_dim = 7
```

Comment:
Hide the global variable?

```suggestion
__input_dim = 7
```

Reply:
it's just a test though - what would you hide it from, and why?